examples/combobox/combobox-autocomplete-both.html

```diff
@@ -426,6 +426,44 @@ <h3 id="rps_label_listbox">Listbox Popup</h3>
           </tr>
         </tbody>
       </table>
+
+      <h3 id="rps_label_button">Button</h3>
+      <table aria-labelledby="rps_label_button rps_label" class="data attributes">
+        <thead>
+          <tr>
+            <th scope="col">Role</th>
+            <th scope="col">Attribute</th>
+            <th scope="col">Element</th>
+            <th scope="col">Usage</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr data-test-id="button-aria-controls">
```

Comment:
I think we need a row before this one for aria-label

Reply:
Fixed